README.md

```diff
@@ -3,19 +3,238 @@
 The Open Policy Agent (OPA) is an open source, general-purpose policy engine
 that enables unified, context-aware policy enforcement across the entire stack.
 
-## Want to learn about OPA?
+## Want to learn more about OPA?
 
 - See [openpolicyagent.org](http://www.openpolicyagent.org) to get started.
 - See [blog.openpolicyagent.org](https://blog.openpolicyagent.org) for blog posts about OPA and policy.
 - Join the conversation on [Slack](http://slack.openpolicyagent.org).
 
 ## Want to get OPA?
 
-See [GitHub releases](https://github.com/open-policy-agent/opa/releases) for binary releases.
+- See [Docker Hub](https://hub.docker.com/r/openpolicyagent/opa/tags/) for Docker images.
+- See [GitHub releases](https://github.com/open-policy-agent/opa/releases) for binary releases and changelogs.
 
-## Want to build OPA?
+## Want to integrate OPA?
 
-See [DEVELOPMENT.md](./docs/devel/DEVELOPMENT.md) for development environment setup and build instructions.
+* See
+  [![GoDoc](https://godoc.org/github.com/open-policy-agent/opa?status.svg)](https://godoc.org/github.com/open-policy-agent/opa/rego)
+  to integrate OPA into services written in Go.
+
+## Want to contribute to OPA?
+
+* See [DEVELOPMENT.md](./docs/devel/DEVELOPMENT.md) to build and test OPA itself.
+* See [CONTRIBUTING.md](./CONTRIBUTING.md) to get started.
+
+## How does OPA work?
+
+OPA gives you a high-level declarative language to author and enforce policies
+across your stack. With OPA, you define rules that govern how your system
+should behave and then you offload policy decisions from your service to OPA.
```

Comment:
Maybe instead of "offload policy decisions" say something that references the integrations discussed above, e.g. "integrate your system with OPA so your system can ask OPA for policy decisions whenever it needs one".

Reply:
Goal is to tell the reader that they've already seen how to integrate their system with OPA, so all they need to know is what kinds of policies they can write and what kinds of queries their system can ask.